php-frontend/src/test/java/org/sonar/plugins/php/api/tests/PhpCheckTestUtils.java

```diff
@@ -63,12 +50,13 @@
  *  $a = 2;  // NOK
  * </pre>
  */
-public class PHPCheckTest {
+public class PhpCheckTestUtils {
```

Comment:
@ivandalbosco btw it's api, we should mention it in API changes http://docs.sonarqube.org/display/PLUG/Custom+Rules+for+PHP